ols/app/metrics.py

```diff
@@ -0,0 +1,13 @@
+"""Prometheus metrics that are exposed by REST API."""
+
+from prometheus_client import Counter, make_asgi_app
+
+# just an example
```

Comment:
just an example but it will establish the precedent that people will copy, so please provide comments explaining what the metrics are intended represent.  In particular it's not clear what the intended distinction is between "call failures" and "validation errors"